pyinstrument/middleware.py

```diff
@@ -16,6 +17,25 @@
     MiddlewareMixin = object
 
 
+def get_renderer_and_extension(path):
+    """Return the renderer instance and output file extension."""
+    if path:
+        try:
+            renderer = import_string(path)()
+        except ImportError as exc:
+            print("Unable to import the class: %s" % path)
+            raise exc
+
+    else:
+        renderer = HTMLRenderer()
+
+    if isinstance(renderer, Renderer):
+        return renderer, renderer.output_file_extension
+    else:
+        print("Renderer should subclass: %s. Using HTMLRenderer" % Renderer)
+        return HTMLRenderer(), HTMLRenderer.output_file_extension
```

Comment:
Pyright was pointing out this can return None. So when the renderer is not a subclass of Renderer, we will be using HTMLRenderer. Let me know if this is fine?

Reply:
I think it would be better to raise an error, instead. I'll update it.

Reply:
I've made the update. For some reason I wasn't able to push to this branch. I've pushed it to https://github.com/joerick/pyinstrument/commit/69b195b6649344b04861df4ca54f8b372a5c88e0 - if you could cherry-pick that here, I think we're good to go.